frontend/src/components/phones/onboarding/RelayNumberPicker.tsx

```diff
@@ -81,6 +82,15 @@ const RelayNumberIntro = (props: RelayNumberIntroProps) => {
 type RelayNumberSelectionProps = {
   registerRelayNumber: (phoneNumber: string) => Promise<Response>;
 };
+type RelayNumberOption = {
+  friendly_name: string;
+  iso_country: string;
+  locality: string;
+  phone_number: string;
+  postal_code: string;
+  region: string;
+};
```

Comment:
This structure is determined directly by the API, right? If so, I'd co-locate it with everything API-related we have, i.e. in `/frontend/src/hooks/api/relayNumber.ts`.